### PR TITLE
Http operators refactor

### PR DESCRIPTION
**Pull request by bannerintheUK**

### CODE REVIEW

File: packages/cerebral-provider-http/src/index.js [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
-export { default as httpPatch } from './factories/httpPatch'
-export { default as httpDelete } from './factories/httpDelete'
+import {
+  httpGet as httpGetFactory,
+  httpPost as httpPostFactory,
+  httpPut as httpPutFactory,
+  httpPatch as httpPatchFactory,
+  httpDelete as httpDeleteFactory} from './operators'
+
+export function httpGet (...args) {
```
Comment (edgesoft) [inline packages/cerebral-provider-http/src/index.js]: Maybe we could log the method as well=

File: packages/cerebral-provider-http/src/index.js [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
+  console.warn('DEPRECATED....')
```
Comment (edgesoft) [inline packages/cerebral-provider-http/src/index.js]: Same here

File: packages/cerebral-provider-http/src/index.js [hunk 3]
```diff
@@ -1,14 +1,40 @@
...
+  return httpGetFactory(...args)
+}
+
+export function httpPost (...args) {
+  console.warn('DEPRECATED....')
```
Comment (edgesoft) [inline packages/cerebral-provider-http/src/index.js]: Same here

File: packages/cerebral-provider-http/src/index.js [hunk 4]
```diff
@@ -1,14 +1,40 @@
...
+  return httpPostFactory(...args)
+}
+
+export function httpPut (...args) {
+  console.warn('DEPRECATED....')
```
Comment (edgesoft) [inline packages/cerebral-provider-http/src/index.js]: Same here

File: packages/cerebral-provider-http/src/index.js [hunk 5]
```diff
@@ -1,14 +1,40 @@
...
+  return httpPutFactory(...args)
+}
+
+export function httpPatch (...args) {
+  console.warn('DEPRECATED....')
```
Comment (edgesoft) [inline packages/cerebral-provider-http/src/index.js]: Same here

File: packages/cerebral-provider-http/src/index.js [hunk 6]
```diff
@@ -1,14 +1,40 @@
...
+  return httpPatchFactory(...args)
+}
+
+export function httpDelete (...args) {
+  console.warn('DEPRECATED....')
```
Comment (edgesoft) [inline packages/cerebral-provider-http/src/index.js]: Same here

File: packages/cerebral-provider-http/src/index.js [hunk 7]
```diff
@@ -102,10 +128,10 @@ export default function HttpProviderFactory (passedOptions) {
             requests[url].xhr.abort()
           })
         },
-        fileUpload (options = {}) {
```
Comment (Guria) [inline packages/cerebral-provider-http/src/index.js]: lets have some deprecation warning for removed method usage

**Pull request merged**